yoti_python_sdk/version.py

```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = "2.8.1"
+__version__ = "2.8.2"
```

Comment:
We should update the requirements.{in,txt} files as well I think?

Reply:
We can do that after it's released to PyPi - as `pip-compile` checks to see if the package is live before generating the `requirements.txt`